login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreen.test.tsx

```diff
@@ -0,0 +1,158 @@
+import { AuthContextProvider } from '../../contexts';
+import { defaultProps as authContextProps } from '../../contexts/AuthContext/AuthContextProvider.test';
+import { BrowserRouter } from 'react-router-dom';
+import { ForgotPasswordScreen } from './ForgotPasswordScreen';
+
+afterEach(cleanup);
+
+describe('Forgot Password Screen tests', () => {
+    let mockOnNext: any;
+    let mockOnPrevious: any;
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    beforeEach(() => {
+        mockOnNext = jest.fn();
+        mockOnPrevious = jest.fn();
+    });
+
+    const renderer = (props?: ForgotPasswordScreenProps): RenderResult =>
+        render(
+            <AuthContextProvider {...authContextProps}>
+                <BrowserRouter>
+                    <ForgotPasswordScreen {...props} />
+                </BrowserRouter>
+            </AuthContextProvider>
+        );
+
+    it('renders without crashing', () => {
+        renderer();
+
+        expect(screen.getByText('Forgot Password')).toBeInTheDocument();
+    });
+
+    it('renders with props crashing', () => {
+        renderer({
+            emailLabel: 'Email Address',
+            initialEmailValue: '',
+            emailValidator: (): any => {},
+            onNext: (): any => {},
+            contactPhone: '',
+            slots: {},
+            slotProps: {},
+        });
+
+        expect(screen.getByText('Forgot Password')).toBeInTheDocument();
+    });
+
+    it('email validation test', () => {
+        const { getByLabelText } = renderer();
+
+        const emailInput = getByLabelText('Email Address');
+        expect(emailInput).toHaveValue('');
+        fireEvent.change(emailInput, { target: { value: 'manojlokesh@eaton.com' } });
+        expect(emailInput).toHaveValue('manojlokesh@eaton.com');
+    });
+
+    it('firing onPrevious Callback functions', () => {
+        renderer({
+            WorkflowCardActionsProps: {
+                canGoPrevious: true,
+                showPrevious: true,
+                previousLabel: 'Back',
+                onPrevious: mockOnPrevious(),
+            },
+        });
+
+        const previousButton = screen.getByText('Back');
+        expect(previousButton).toBeInTheDocument();
+        fireEvent.click(previousButton);
+        expect(mockOnPrevious).toHaveBeenCalled();
+    });
+
+    it('firing onNext Callback functions', () => {
+        const { getByLabelText } = renderer({
+            WorkflowCardActionsProps: {
+                canGoNext: true,
+                showNext: true,
+                nextLabel: 'Next',
+                onNext: mockOnNext(),
+            },
+        });
+
+        const emailInput = getByLabelText('Email Address');
+        const nextButton = screen.getByText('Next');
+        expect(emailInput).toHaveValue('');
+        expect(screen.getByText(/Next/i)).toBeDisabled();
+        fireEvent.change(emailInput, { target: { value: 'manojlokesh@eaton.com' } });
```

Comment:
can we replace all the entries for manojlokesh@eaton.com with a fake email of some kind. unless he wants spam